### DISCUSSION TITLE
Ambiente di STAGE

### DISCUSSION THREAD

**Discussion by stefanosilvestrinimia** [fattureincloud/api#336] [Q&A] [answered]
buongiorno, per avere un ambiente di STAGE devo creare una nuova azienda e quindi pagare un abbonamento per quella azienda? vorrei evitare di sporcare l'ambiente di PROD (account a pagamento).

grazie mille

**Comment by emajo** [comment, marked as answer]
Ciao @stefanosilvestrinimia, attualmente non disponiamo di [account specifici per gli sviluppatori](https://developers.fattureincloud.it/docs/developer-account#-trial-extension), per cui il nostro consiglio è quello di creare un nuovo account in trial e, eventualmente, è possibile contattare il nostro supporto clienti che andrà ad estendere il periodo di prova, ovviamente specificando che ciò è necessario per lo sviluppo di un integrazione.

Grazie,
Emanuele.

**Reply by stefanosilvestrinimia** [reply]
grazie mille
